src/cli/plugin/setting_parser.js

```diff
@@ -84,8 +84,12 @@ export default function createSettingParser(options) {
       settings.package = parts.shift();
     }
 
-    if (!settings.action || (options.install && options.remove)) {
-      throw new Error('Please specify either --install or --remove.');
+    if (options.list) {
+      settings.action = 'list';
+    }
+
+    if (!settings.action || (options.install && options.remove && options.list)) {
```

Comment:
I think this conditional needs to get a bit more robust.  It seems to be verifying that the user isn't trying to specify conflicting commands, which can be represented with a single conditional when there are only two possibilities.  Now that there are three possibilities, I assume we want to verify that the user isn't trying to perform *any combination* of those commands, which will take more than a simple conditional to verify.